.env.example

```
@@ -0,0 +1,22 @@
+BETTER_AUTH_SECRET=your-better-auth-secret
+BETTER_AUTH_URL=http://localhost:3000 # Base URL of your app
+
+DATABASE_URL='GET-ONE-FROM-NEON-PROJECT-DASHBOARD'
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)